bus:
  - can0
loop_frequency: 650

cycle_time_error_threshold: 1
thread_priority: 30

actuators:
  motor1:
    bus: can0
    id: 0x207
    type: rm_2006
